pytorch/SymmetryFunctions.py

```diff
@@ -61,12 +63,13 @@ class TorchANISymmetryFunctions(torch.nn.Module):
 
         >>> print(energy, forces)
     """
+    holder: Optional[torch.classes.NNPOps.CustomANISymmetryFunctions]
```

Comment:
Maybe call it something more descriptive like `symmetryFunctions`?  In my example, the name `holder` was a `ValueHolder` object.

Reply:
I'm still working on cleaning up the code, but to call an instance `symmetryFunctions` is misleading and might be confused with an actual reference to the function `symFunc`.